helpers/voting/makePriceRequestsByKey.ts

```diff
@@ -74,6 +74,29 @@ function formatPriceRequest(
     : priceRequest.isGovernance;
   const rollCount = priceRequest.rollCount || 0;
   const resolvedPriceRequestIndex = priceRequest.resolvedPriceRequestIndex;
+  const isAssertion = ["assertionId:", "ooAsserter:", "childChainId:"].every(
```

Comment:
adds assertion meta data to vote type, could probably add the osnap stuff as well